config/core/test-configs-chromeos.yaml

```diff
@@ -108,14 +187,62 @@ test_plans:
       <<: *cros-tast-params
       base_template: 'chromeos/cros-boot-qemu.jinja2'
 
-  cros-tast_qemu-fixed:
+  cros-tast_qemu-kernel:
+    <<: *cros-tast-qemu
+    params: 
+      <<: *cros-tast-qemu-params
+      tests: *cros-tast-kernel-tests
+
+  cros-tast_qemu-perf:
+    <<: *cros-tast-qemu
+    params: 
+      <<: *cros-tast-qemu-params
+      tests: *cros-tast-perf-tests
+
+  cros-tast_qemu-platform:
+    <<: *cros-tast-qemu
+    params:
+      <<: *cros-tast-qemu-params
+      tests: *cros-tast-platform-tests
+
+  cros-tast_qemu-video:
+    <<: *cros-tast-qemu
+    params:
+      <<: *cros-tast-qemu-params
+      tests: *cros-tast-video-tests
+
+  cros-tast_qemu-fixed: &cros-tast_qemu-fixed
     <<: *cros-tast-qemu
     base_name: cros-tast-fixed
     filters: *cros-qemu-filters-fixed
-    params:
+    params: &cros-tast_qemu-fixed-params
       <<: *cros-tast-qemu-params
       fixed_kernel: true
 
+  cros-tast_qemu-fixed-kernel:
+    <<: *cros-tast_qemu-fixed
+    params:
+      <<: *cros-tast_qemu-fixed-params
+      tests: *cros-tast-kernel-tests
+
+  cros-tast_qemu-fixed-perf:
+    <<: *cros-tast_qemu-fixed
+    params:
+      <<: *cros-tast_qemu-fixed-params
+      tests: *cros-tast-perf-tests
+
+  cros-tast_qemu-fixed-platform:
+    <<: *cros-tast_qemu-fixed
+    params:
+      <<: *cros-tast_qemu-fixed-params
+      tests: *cros-tast-platform-tests
+
+  cros-tast_qemu-fixed-video:
+    <<: *cros-tast-fixed
```

Comment:
Tiny typo:
<<: *cros-tast_qemu-fixed

Reply:
Otherwise it crash some qemu tests on staging:
https://bot.staging.kernelci.org/job/chromeos/job/test-runner/7088/console
My debugging shows that:
```
qemu_x86_64-uefi-chromeos
cros-tast_qemu-fixed-video
Traceback (most recent call last):
  File "/home/nuclearcat/Documents/kernelci-core/./kci_test", line 319, in <module>
    status = opts.command(configs, opts)
  File "/home/nuclearcat/Documents/kernelci-core/./kci_test", line 262, in __call__
    job = lab_api.generate(params, device_config, plan_config,
  File "/home/nuclearcat/Documents/kernelci-core/kernelci/lab/lava/__init__.py", line 88, in generate
    data = template.render(params)
  File "/usr/local/lib/python3.9/dist-packages/Jinja2-3.0.3-py3.9.egg/jinja2/environment.py", line 1291, in render
    self.environment.handle_exception()
  File "/usr/local/lib/python3.9/dist-packages/Jinja2-3.0.3-py3.9.egg/jinja2/environment.py", line 925, in handle_exception
    raise rewrite_traceback_stack(source=source)
  File "config/lava/chromeos/cros-tast.jinja2", line 1, in top-level template code
    {% extends base_template|default('chromeos/cros-boot.jinja2') %}
  File "config/lava/chromeos/cros-boot.jinja2", line 1, in top-level template code
    {% extends 'base/kernel-ci-base.jinja2' %}
  File "config/lava/base/kernel-ci-base.jinja2", line 83, in top-level template code
    {% block actions %}{% endblock %}
  File "config/lava/chromeos/cros-tast.jinja2", line 3, in block 'actions'
    {{ super() }}
  File "config/lava/chromeos/cros-boot.jinja2", line 24, in block 'actions'
    url: {{ cros_flash_kernel.base_url }}{{ cros_flash_kernel.image }}
  File "/usr/local/lib/python3.9/dist-packages/Jinja2-3.0.3-py3.9.egg/jinja2/environment.py", line 474, in getattr
    return getattr(obj, attribute)
jinja2.exceptions.UndefinedError: 'cros_flash_kernel' is undefined
```

Reply:
names and plans order corrected